ux.symfony.com/assets/styles/demos/live-memory/app/_font.scss

```diff
@@ -0,0 +1,25 @@
+// -----------------------------------------------------------------
+// Font(s)
+// -----------------------------------------------------------------
+
+@font-face {
```

Comment:
I think these fonts are already available via a different `_font.scss`, right?

Reply:
I needed another size of ubuntu.. but as the demo is now only one screen, and no text content will be posted there in a forseeable future... i remove entirely the fonts :)